modules/preview-react/radio/stories/radio.stories.mdx

```diff
@@ -0,0 +1,133 @@
+import {Meta} from '@storybook/addon-docs';
+import {Specifications, SymbolDoc} from '@workday/canvas-kit-docs';
+
+import {RadioGroup2} from '@workday/canvas-kit-preview-react/radio';
+
+import {Basic} from './examples/Basic';
+import {Alert} from './examples/Alert';
+import {Error} from './examples/Error';
+import {Inverse} from './examples/Inverse';
+import {LabelPosition} from './examples/LabelPosition';
+import {NoValue} from './examples/NoValue';
+import {RefForwarding} from './examples/RefForwarding';
+import {Disabled} from './examples/Disabled';
+import {Required} from './examples/Required';
+
+<Meta title="Preview/Radio" component={RadioGroup2} />
+
+# Canvas Kit Radio
+
+Radio Buttons allow a user to select one value from a predefined list of 7 or fewer options.
+
+[> Workday Design Reference](https://design.workday.com/components/inputs/radio-buttons)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+### Basic Example
+
+`Radio` buttons are intended to be grouped together using a wrapping `RadioGroup` element with a
+`name` attribute. This will ensure that all `<input type="radio" />` will share the same `name`,
+grouping them together.
```

Comment:
Love all the detail in the `Basic Example` section! We chatted earlier about reorganizing this content so it reads less like a list of independent thoughts. I'd recommend grouping these sentences into paragraphs like so:

> `Radio` buttons are intended to be grouped together using a wrapping `RadioGroup` element with a `name` attribute. This will ensure that all `<input type="radio" />` will share the same `name`, grouping them together. Also, each `RadioGroup.Radio` must have a unique `value`. This is used to determine which radio has been selected. Additionally, you can provide an `initialValue` to the `RadioGroup` element so that a value is pre selected from the group. To tie it all together, provide an `onChange` to `RadioGroup` element that sets the selected value on [state](https://react.dev/learn/state-a-components-memory).
>
> `RadioGroup` should be used in tandem with [Form Field](/components/inputs/form-field/) where the `useFieldset` prop is set to `true` to meet accessibility standards. This ensures that the `label` text from `FormField` is attached to the `RadioGroup` and read out as a group for voiceover.
>
> `<ExampleCodeBlock code={Basic} />`
>
> Because the `RadioGroup` and `RadioGroup.Radio` have the correct WAI-ARIA
[properties](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radiogroup_role#associated_wai-aria_roles_states_and_properties), keyboard navigation is supported.

The first paragraph introduces the reader to the component and tells them how to set the values of the Radio buttons and read the selected value of the RadioGroup. The second paragraph layers FormField on top of things. I opted to put the example next since I felt the two preceding paragraphs provided enough foundation. I included the bit about keyboard navigation after the example since it felt more supplementary.

I copied your sentences verbatim in my suggestion above and just reordered them. However, I think we can also remove some filler in there to tidy things up. For example:

> Additionally, you can provide an initialValue to the RadioGroup element so that a value is pre selected from the group.

...could be...

> You may provide an initialValue to the RadioGroup to pre-select a value

I'll leave the rest as an exercise for you 😄 We can cover further tweaks in our next pass.